config/helm/chart/default/templates/Common/csi/clusterrole-csi.yaml

```diff
@@ -37,30 +29,6 @@ rules:
       - create
```

Comment:
did you check if these need to be different from other places we use events?

Reply:
you are referring to the weird `create/patch` in both clusterRole and none-clusterRoles?

it makes little sense: (I tested it now that you asked)
- Operator: Only creates/updates/patch (don't know the exact "verbs") in its own namespace, so the clusterRole is not necessary
    - purpose: needed for leader-election
- Webhook: Only creates/updates/patch (don't know the exact "verbs") in its own namespace so the clusterRole is not necessary
    - purpose: we create events when we inject or fail to inject a pod (only in case of oneagent injection)
    - seems kinda useless 😬 
- CSI-driver: Only creates/updates/patch (don't know the exact "verbs") in its own namespace
    - purpose: we create events when download a given version
    - seems kinda useless 😬

Reply:
thanks for testing that. I had an inkling they wouldn't be ncessary 😄 

we can just remove them here as well then!

Reply:
If we can remove verbs from the CSI driver roles/clusterroles let's do that and keep the rest as is for now.

Reply:
related change [Move event permissions to role from clusterRole](https://github.com/Dynatrace/dynatrace-operator/pull/3668/commits/cfdbfad917dc660a7ddf47dc547475d5adeb7391)